bridge/engine/EngineMixer.cpp

```diff
@@ -84,7 +84,7 @@ EngineMixer::EngineMixer(const std::string& id,
     assert(audioSsrcs.size() <= SsrcRewrite::ssrcArraySize);
     assert(videoSsrcs.size() <= SsrcRewrite::ssrcArraySize);
 
-    std::memset(_mixedData, 0, samplesPerFrame20ms * channelsPerFrame);
+    std::memset(_mixedData, 0, samplesPerFrame20ms * channelsPerFrame * sizeof(int16_t));
```

Comment:
I think it is safer this way:

```suggestion
    std::memset(_mixedData, 0, sizeof(_mixedData));
```